CONTRIBUTING.md

```diff
@@ -19,6 +19,8 @@ You need to install the following dependencies:
     To install specific Rust version, use `rustup install 1.26.2` command.
   * The [system dependencies](https://exonum.com/doc/get-started/install/) of Exonum. 
   You do _not_ need to manually fetch and compile Exonum.
+  * For automatic packaging of the EJB App you also need `CMake` installed in your system. 
```

Comment:
Is it implicit dependency? I haven't seen cmake being used anywhere.

Reply:
It's required by `cargo-license` tool :confused: